src/Illuminate/Foundation/Console/ModelMakeCommand.php

```diff
@@ -50,6 +50,11 @@ public function fire()
 	 */
 	protected function getStub()
 	{
+		if ( ! empty($this->laravel['config']['console.stubs.model.default']))
```

Comment:
Why not use the get method on the config object with the 2nd param being `__DIR__.'/stubs/model.stub'` to avoid all of this logic?